### PR TITLE
Choose whitelisted IP in public scenario

### PR DESCRIPTION
**Pull request by gabrielgrigorie**
Choose whitelisted IP in public scenario:
-added setting in firewall rules pg_whitelisted_ip
-tags and variables new version 1.4.1
-schema.yaml file equipped withe the new whitelisted ip rule

### CODE REVIEW

**Review by lfeldman — COMMENTED** [review]
In remote.tf please replace `vat `with `var` (line 11)

File: remote.tf [hunk 1]
```diff
@@ -8,6 +8,7 @@ data "template_file" "postgresql_install_binaries_sh" {
     pg_password       = var.postgresql_password
     pg_version_no_dot = replace(var.postgresql_version, ".", "")
     pg_version        = var.postgresql_version
+    pg_whitelisted_ip = vat.pg_whitelisted_ip
```
Comment (lfeldman) [inline remote.tf:11]: please replace `vat` with `var`

File: schema.yaml [hunk 2]
```diff
@@ -99,6 +99,16 @@
         visible: true
         default: false
 
+      pg_whitelisted_ip: 
+        type: string
+        visible: 
+           not:
+             -create_in_private_subnet
```
Comment (lfeldman) [inline schema.yaml:106]: Please add the space between dash sign (-) and the variable `create_in_private_subnet.`

File: variables.tf [hunk 3]
```diff
@@ -13,6 +13,10 @@ variable "use_existing_vcn" {
   default = false
 }
 
+variables "pg_whitelisted_ip" {
+  defatul = ""
```
Comment (lfeldman) [inline variables.tf:17]: Please replace defatul to default.

**Review by gabrielgrigorie — COMMENTED** [review]
added the space between dash sign (-) and the variable create_in_private_subnet.
replaced defatul to default.
replaced vat with var

**Review by lfeldman — APPROVED** [review]

**Pull request merged**